app/main.py

```diff
@@ -2,0 +1,12 @@
+
+def copy_file(command: str) -> None:
+    date = command.split()
+    if date[0] != "cp":
+        return
+    first_txt, second_txt = date[1], date[2]
+    if first_txt == second_txt:
+        return
+    else:
+        with (open(first_txt, "r") as file_in,
```

Comment:
Consider using more descriptive variable names for the file objects. For example, `source_file` and `destination_file` would be more descriptive than `file_in` and `file_out`.